projects/Mallard/src/helpers/event-emitter.ts

```diff
@@ -0,0 +1,5 @@
+import events from 'events'
```

Comment:
As explained in our meeting. This event emitter is designed to let the Issue Summary Provider know that the edition has changed. This is because it lives in "Apollo Land" and we need to let the resolver know when a change happens so it can refetch and display the fronts